README.md

```diff
@@ -1,3 +1,13 @@
 # NRSS - RSS Feeds for NRK sin podcaster  
 
 https://nrss.deno.dev/
+
+## Local devlopment
+install Deno
+https://deno.land/manual/getting_started/installation
+
+### Run the application
+``` shell
+deno run main.ts
```

Comment:
```suggestion
deno task start
```

This change uses the task in `deno.json`, meaning that it'll also update on file changes.